modules/publicize/publicize.php

```diff
@@ -245,33 +407,146 @@ function show_options_popup( $service_name, $connection ) {
 		return false;
 	}
 
-	function user_id() {
-		global $current_user;
-		return $current_user->ID;
+	/**
+	 * Whether the Connection is "valid" wrt Facebook's requirements.
+	 *
+	 * Must be connected to a Page (not a Profile).
+	 * (Also returns true if we're in the middle of the connection process)
+	 *
+	 * @param object|array The Connection object (WordPress.com) or array (Jetpack)
+	 * @return bool
+	 */
+	function is_valid_facebook_connection( $connection ) {
```

Comment:
Could absorb the check whether we're dealing with a Facebook connection, and thus generalized to `is_valid_connection()` (to possibly hold other checks later).

Reply:
That makes sense. I guess let's wait to change it when we have that need, though?